homeassistant/components/lutron/light.py

```diff
@@ -63,6 +100,16 @@ def brightness(self) -> int:
 
     def turn_on(self, **kwargs: Any) -> None:
         """Turn the light on."""
+        if self._is_fan:
+            async_create_issue(
+                self.hass,
+                DOMAIN,
+                "deprecated_light_fan_on",
+                breaks_in_ha_version="2024.7.0",
+                is_fixable=False,
```

Comment:
Repair issues for deprecated service calls should be fixable and persistent. Then the user can fix them when they have updated their automations and scripts.

Reply:
done